frontend/src/app/models/group.ts

```diff
@@ -0,0 +1,20 @@
+export class Group {
+  groupID: string;
+  projectID: string;
+
+  name: string;
+  members: string[];
+}
+
+export class User {
```

Comment:
This and Role are already defined inside models/user.ts

Also you should use AuthUser from there since it hides the password.